config/printer-anet-et4x-2020.cfg

```diff
@@ -0,0 +1,225 @@
+# This file contains common pin mappings for the Anet ET4X printer.
+# To use this config, the firmware should be compiled for the
+# STM32F407. When running "make menuconfig", enable "extra low-level
+# configuration setup", select (no bootloader), disable "USB for
+# communication".
+
+# The "make flash" command does not work on the Anet ET4. Instead,
+# after running "make", copy the generated "out/klipper.bin" file to a
+# file named "project.bin" on an SD card and then restart the Alfawise
+# with that SD card.
+
+[stepper_x]
+step_pin: PB6
+dir_pin: !PB5
+enable_pin: !PB7
+step_distance: .0125
```

Comment:
FYI, `step_distance` is deprecated.  All new example configs should use `rotation_distance`.

Reply:
will update.